main.nf

```diff
@@ -244,16 +244,16 @@ process picard {
     module 'picard/2.0.1'
     module 'BEDTools'
     module 'samtools'
-    
+
     cpus 2
     memory { 16.GB * task.attempt }
     time { 24.h * task.attempt }
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
     maxRetries 3
     maxErrors '-1'
-    
+
     publishDir "${params.outdir}/picard", mode: 'copy'
-    
+
     input:
     file bam_picard
     set val(prefix) from name_for_picard
```

Comment:
Yup, that worked! Looks good now, but there was another bug that was different which needed to be fixed after splitting the process into two.